user-guide/Standard_Apps/PA/Advanced_Configuration/PA_Integrating_Gateways.md

```diff
@@ -35,6 +35,69 @@ The *PA GW RoutingRules* parameter of a profile instance can contain a set of or
+                "OutgoingInterfaceIds": [],
+                "DoTagTokens": true,
+                "Conditions": [{
+                    "IncomingGatewayKeyName": "*",
+                    "Value": "*",
+                    "Operation": "any",
+                    "IncomingInterfaceId": -1
+                },
+                "DomField": {
+                    "FieldDescriptorId": "9a15783b-ccc0-4d8f-9574-0870c55f4a73",
+                    "SectionDefinitionId": "5d6c39ed-f616-4ad2-b311-28e39d52505a"
+                 }
+            ]
+        }
+    ]
+}
+```
+
+### Routing Rule Json Object
+
+#### RoutingRule Object
+
+| Name | Type | Value | Information |
+| --- | --- | --- | --- |
+| Evaluation | string (discrete) | First Matching <br>or All Matching | First Matching: The evaluation will stop when the first rule has a match. <br>All Matching: All rules will be evaluated. Only triggered if all of the conditions match. |
+| Rules | [Rule](#rule-object)[] | [Rule](#rule-object) | |
+
+#### Rule Object
+
+| Name | Type | Value | Information |
+| --- | --- | --- | --- |
+| SequenceId | int |  | Needs to be unique. |
+| Description | string |  | Description for developers. |
+| OutgoingInterfaceIds | int[] | 11, 12, 13, 14, 15 | The list of all the output interface IDs that this rule has to generate a token for. These represent the outgoing interfaces that need to be triggered when this rule matches. |
+| DoTagTokens | bool | true or false | |
+| Conditions | [RoutingCondition](#routingcondition-object)[] | [RoutingCondition](#routingcondition-object) | |
+
+#### RoutingCondition Object
+
+| Name | Type | Value | Information |
+| --- | --- | --- | --- |
+| IncomingGatewayKeyName | nullable string | "\*", null, or another value | For null or "*": Any incoming gateway key name.<br>If you use a DomField, this is entirely ignored, so this will most likely be null.|
+| Value | string | | The value that will be used to validate a DOM Field against. |
+| Operation | string (discrete) | any or equal | When operation is **any**, the incoming gateway key name and value are ignored.<br>When operation is **equal**, it will match when the incoming value matches the *Value* field. <br> When a DomField is given, it will check the DOM Field instead of the incoming gateway key name. |
+| IncomingInterfaceId | int | -1, 1, 2, 3, 4, 5 | It can be one of the specified values. -1 represents all interfaces. |
+| DomField | [IncomingDomField](#incomingdomfield-object) | | |
+
+#### IncomingDomField Object
+
+| Name | Type | Value | Information |
+| --- | --- | --- | --- |
+| FieldDescriptorId | string | | The GUID of the field descriptor defined in the Section Definition of the DOM definition. |
+| SectionDefinitionId | string | | The GUID of the Section definition where the field descriptor belongs to. |
+| UserDomInstance | string | | The name of the user DOM instance. When this is defined, it will process a user DOM instance instead of the Process DOM instance. |
```

Comment:
The label of the Activity from the Process Definition where the user task gets generated. When this is defined...